### PR TITLE
Added new folder via upload (Sanskrit resources)

### PR DESCRIPTION
**Pull request by radheesh1**

### CODE REVIEW

**Comment by vvasuki** [comment]
Thanks for the contribution.

- Please place these files in a folder called `teaching_aids` at https://github.com/sanskrit-coders/sanskrit-coders.github.io/tree/hugo-source/content/content . Additionally, add an `_index.md` file there (You can copy https://github.com/sanskrit-coders/sanskrit-coders.github.io/blob/hugo-source/content/content/_index.md and modify it's contents as appropriate.)
- The third pdf  can be plain text. Please add it's contents as a markdown file named: `course_management_systems.md` under the `teaching_aids` folder mentioned above.

**Comment by radheesh1** [comment]
> Thanks for the contribution.
> 
> * Please place these files in a folder called `teaching_aids` at https://github.com/sanskrit-coders/sanskrit-coders.github.io/tree/hugo-source/content/content . Additionally, add an `_index.md` file there (You can copy https://github.com/sanskrit-coders/sanskrit-coders.github.io/blob/hugo-source/content/content/_index.md and modify it's contents as appropriate.)
> * The third pdf  can be plain text. Please add it's contents as a markdown file named: `course_management_systems.md` under the `teaching_aids` folder mentioned above.

dhanyawaadaH mahoday

We are volunteers at Sanskrit Club IITR (Indian Institute of Technology - Roorkee).
We are organizing the world's first Samskritam hackathon by the title TARGET Samskritam Hackathon. You may find more details here - https://www.iitr.ac.in/sanskritclub/target.html.
We got your reference from one of the institutions collaborating with us.

We would like to upload more such resources which we are collecting from those willing to contribute towards this event, in your repository. We have included this repository in our website, the link of which has been provided above, so that the participants may have easy access to these resources.

dhanyawaadaH
Radheesh
Sanskrit Club IITR

**Comment by vvasuki** [comment]
Thanks for the update. Copies of the files placed in the earlier 'Sanskrit resources' folder need to be deleted (being duplicates and in the wrong place).

**Comment by radheesh1** [comment]
created teaching_aids folder and added resources

**Comment by radheesh1** [comment]
> Thanks for the update. Copies of the files placed in the earlier 'Sanskrit resources' folder need to be deleted (being duplicates and in the wrong place).

dhanyawaadaH 
followed your instructions and deleted previous files.

File: content/content/teaching_aids/course_management_systems.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Samskrit Bharti Courses Management System
```
Comment (vvasuki) [inline content/content/teaching_aids/course_management_systems.md:1]: Please change heading to `##`. You may need to adjust subsection levels accordingly. Also, please ensure that there are no spaces before #

**Comment by vvasuki** [comment]
(ping)

**Pull request merged**